cigaradvisor/blocks/carousel/carousel.js

```diff
@@ -86,15 +86,15 @@ export default async function decorate(block) {
       if (currentIndex < (items.length - itemsToShow)) {
         currentIndex += 1;
         slidesWrapper.style.transform = `translate3d(-${currentIndex * offset}%, 0, 0)`;
-        block.querySelector('.arrow-prev').style.display = 'inline-block';
+        block.querySelector('.arrow-prev').style.display = 'flex';
```

Comment:
we shouldn't set literal styles on the elements. Should add a class which will set `display: none` as an override to the `display: flex` on the element.